IQSS/dataverse-frontend
# Add Breadcrumb to Create Dataset Page

## What this PR does / why we need it:
In Create Dataset page, we don't have the breadcrumb for the page which is supposed to have.

## Which issue(s) this PR closes:


- Closes #480 

## Special notes for your reviewer:
Changes:
1. Add breadcrumb to the page
2. Create a createDatasetSkeleton for loading state
3. Add containers to multiple 'Create Dataset' text existing for E2E testing(Collection.spec.ts)

## Suggestions on how to test this:
1. Log in to the SPA
2. Locate and click on the "Add Data" button in the navigation bar
3. From the dropdown or menu that appears, select "New Dataset"
5. Check the breadcrumb trail at the top of the page. It should display: "Root > Create Dataset"

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:
Yes, breadcrumb added.
![image](https://github.com/user-attachments/assets/72b2b66b-ba3f-4017-9ff2-fb8b2f8fbf52)

## Is there a release notes update needed for this change?:
No
## Additional documentation:
No
